shared/src/main/tut/README.md

```diff
@@ -709,6 +709,123 @@ The `UnitGroup` values provided with Squants are only samples and aren't intende
 We encourage users to make their own `UnitGroup` defintitions and submit them as PRs if they're generally
 applicable.
 
+## Formatters
+
+Squants provides an experimental API for formatting Quantities in the "best unit." For example, 
```

Comment:
I wonder a bit about the `experimental` part. If this is something we may change then the `experimental` tag fits very well but then perhaps it should be on an `experimental` package.

Do you foresee possible changes in the near future that could break the API?